components/supervisor/frontend/src/index.ts

```diff
@@ -153,7 +153,7 @@ const toStop = new DisposableCollection();
                 // reload the page if the workspace was restarted to ensure:
                 // - graceful reconnection of IDEs
                 // - new owner token is set
-                window.location.href = startUrl.toString();
+                window.location.href = startUrl.with({ search: "error=true" }).toString();
```

Comment:
`error=true` looks strange here though 😬 

Does it make sense to check whether conditions of stopped event were erroneous?

Reply:
> error=true looks strange here though

Agreed. :laughing: Do you have a better name?

@jankeromnes Suggested removing this redirect alltogether. It _might_ be not necessary anymore, as we don't have the "auto-restart" expectation anymore. But I'm unsure about all edge-cases: https://github.com/gitpod-io/gitpod/pull/8125#issuecomment-1034734273

> Does it make sense to check whether conditions of stopped event were erroneous?

This is not sufficient, as we don't know which instance to look at. ATM `dontAutostart` basically looks at the "last", while before we'd always look at the "last running (or created on if there was none)".

Reply:
> Does loading screen now reload the page?

No, it does not. 

> Otherwise it is going to leave a user with empty page without the loading screen and IDE.

:+1:

Reply:
Although it seems to break now if i restart a perfectly stopped workspace from the dashboard as well. It reloads and then hangs in `opennnig workspace...` and then times out without ever revealing an IDE.

Reply:
Hm, that's strange for sure. That's one of the standard cases... Will re-test and debug again. :thinking: :pray:

Reply:
Maybe it is alright to remove. ping me if you cannot reproduce it.

Reply:
how about `not-found` instead of `error`?